optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -33,8 +33,8 @@ that deliver a near-optimal solution in a reasonable amount of time.
 
 == What you'll need
 
-* https://www.oracle.com/java/technologies/javase-downloads.html[JDK 8 or later]
-* https://maven.apache.org/download.cgi[Maven 3.2+] or https://gradle.org/install/[Gradle4+]
+* https://jdk.java.net/[JDK 11 or later]
```

Comment:
Let's link to https://adoptopenjdk.net/ like we do in the quickstart readme's.

Motivation: Oracle's JDK downloads are non-free or free-for-a-limited-period AFAIK.
The AdoptOpenJDK project is where the community is working around that.

Note: AdoptOpenJDK is in the process of being renamed to Adoptium, but we will only update our links from adoptopenjdk.net to adoptium.net once that renaming is complete. Meanwhile, we need to link to https://adoptopenjdk.net/ so we can find-in-path it easily later this year.